src/test/java/io/stargate/sgv2/jsonapi/fixtures/testdata/LogicalExpressionTestData.java

```diff
@@ -166,9 +222,51 @@ public static Object value(DataType type) {
         return "text-value";
       }
       if (type.equals(DataTypes.DURATION)) {
-        // we handle duration as a string until it gets to the codec
-        return "P1H30M";
+        return "P1H30M"; // Handle duration as a string
+      }
+      if (type.equals(DataTypes.INT)) {
+        return 25;
+      }
+      if (type.equals(DataTypes.BIGINT)) {
+        return 1000000L;
+      }
+      if (type.equals(DataTypes.DECIMAL)) {
+        return 19.99;
```

Comment:
Doesn't `DECIMAL` map to `BigDecimal`, instead of `Double`? (constant `19.99` is `double` and gets boxed as `Double).
That may be fine, just double-checking.

Reply:
Thanks, fixed

Reply:
fixed